### PR TITLE
some interesting crates from bevy's ecosystem this month

### PR DESCRIPTION
**Pull request by ChristopherBiscardi**
_Part of #1523_

I've added a few interesting crates from this month to the issue, as requested. I intentionally avoided crates @janhohenheim and I already discussed (like bevy_lunex) or were already in the issue and prioritized crates that seemed more generally interesting to the wider gamedev community.

### CODE REVIEW

File: content/news/052/berdicles.avif
Comment (Vrixyz) [inline content/news/052/berdicles.avif:1]: avif file is surprising, I'm just noting because I'm not used to the format.

**Comment by janhohenheim** [comment]
Note for other reviewers: this is past the submission deadline, but I explicitly asked Chris on Discord if I had missed any cool crates, so treat this as part of the editorial process :)

**Comment by janhohenheim** [comment]
Did a little editorial pass and also added @JMS55's post as well :)

File: content/news/052/berdicles.avif
Comment (janhohenheim) [reply content/news/052/berdicles.avif:1]: I switched it to jpeg, which was smaller

**Review by janhohenheim — APPROVED** [review]

**Pull request merged**